source/stylesheets/refills/unstyled/_tooltip.scss

```diff
@@ -0,0 +1,36 @@
+.tooltip-item {  
+  display: inline-block;
+  position: relative;
+  text-align: center;
+
+  &:focus,
+  &:hover .tooltip {
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .tooltip {
+    @include position(absolute, null 0 3em 0);
+    @include transition (all 0.2s ease-in-out);
+    background-color: #f5f5f5;
```

Comment:
Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.